.gitignore

```diff
@@ -1,16 +1,14 @@
 **/*.log
 **/node_modules
 .coverage
+coverage
 .nyc_output
 docs/
 out/
 build/
 system-test/secrets.js
 system-test/*key.json
 *.lock
-.vscode
-**/package-lock.json
 .DS_Store
-google-cloud-logging-winston-*.tgz
-google-cloud-logging-bunyan-*.tgz
+package-lock.json
```

Comment:
I think we do still want `.vscode` (that's not getting checked in, right?) and `**/package-lock.json`, because the samples install their own `node_modules`.

Reply:
I will put it back.

Reply:
@feywind synthtool remove `.vscode` and `**/package-lock.json`automatically

Reply:
@alexander-fenster Does this synthtool change seem right? Maybe we're excluding `.vscode` and `**/package-lock.json` elsewhere?

Reply:
@bcoe what do you think? It might make sense to exclude `.vscode` and nested lockfiles.